network/forks/fork.go

```diff
@@ -40,10 +40,7 @@ func IsForkNextEpoch(genesisTime time.Time, genesisValidatorsRoot []byte) (bool,
 
 // ForkDigestFromEpoch retrieves the fork digest from the current schedule determined
 // by the provided epoch.
-func ForkDigestFromEpoch(currentEpoch types.Epoch, genesisValidatorsRoot []byte) ([4]byte, error) {
-	if len(genesisValidatorsRoot) == 0 {
```

Comment:
this is probably going to break a ton of tests, you would be better of removing it here

Reply:
I did remove the check. Did you mean something else?